<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DePropertyEditWidget</class>
 <widget class="QWidget" name="DePropertyEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DePropertyEditWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit
{
border:1px solid gray;
border-radius:5px;
min-height:30px;
font: 16pt &quot;新宋体&quot;;
}

QLabel
{
font: 14pt &quot;新宋体&quot;;
min-width:60px;
}

QGroupBox {
     border: 1px solid gray;
     border-radius: 3px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 10px;
     margin:10px;
     font: 12pt &quot;新宋体&quot;;
 }

 QGroupBox::title {     
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

QPushButton {
    
	font: 16pt &quot;新宋体&quot;;
	background-color: rgb(244, 244, 244);
    border: 1px solid #8f8f91;
    border-radius: 6px;
    min-width: 100px;
    min-height: 38px;
 }

 QPushButton:pressed {
	background-color: rgb(255, 255, 255);
 }
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>391</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="DeVirtualKeyboradWidget" name="virtualBoradWidget" native="true"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">max-width:280px;
max-height:100px;</string>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit">
           <property name="maxLength">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeVirtualKeyboradWidget</class>
   <extends>QWidget</extends>
   <header location="global">devirtualkeyboradwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
